### DISCUSSION TITLE
GitHub official Youtube channel

### DISCUSSION THREAD

**Discussion by Eprince-hub** [community/community#53024] [Other Feature Feedback, Questions, & Ideas]
We have been using this youtube channel [GitHub Training & Guides](https://web.archive.org/web/20230304072801/https://www.youtube.com/@GitHubGuides) to teach our students Git and GitHub as it has easy-to-follow and important concepts of GitHub tutorials.
 
Recently the youtube channel has not been available and seems to have been deleted. I want to know if anyone knows about this channel and what happened to it. Is it permanently closed and deleted, or just have moved to another name?

I couldn't find these videos from the [official GitHub youtube channel](https://www.youtube.com/user/github)
[Getting Started with GitHub](https://www.youtube.com/watch?v=noZnOSpcjYY)
[Get Up and Running](https://www.youtube.com/watch?v=noZnOSpcjYY&list=PLg7s6cbtAD15G8lNyoaYDuKZSKyJrgwB-)

Does anyone know anything about this youtube channel?

**Comment by neutrinomuon** [comment]
Hi, @Eprince-hub, Thanks for sharing the info. It seems that the GitHub Training & Guides channel and videos were either deleted or made private. However, there are some tutorials you can find on the GitHub documentation website:

https://docs.github.com/en/get-started/quickstart/git-and-github-learning-resources

I think in the Git and GitHub Learning Resources page on the GitHub documentation site provides various resources including videos, articles, and interactive learning tools to help beginners learn Git and GitHub. 

Maybe this will help you. Please, let me know

**Reply by Eprince-hub** [reply]
Thanks a lot for the response, We've linked to a lot of the resources from the GitHub documentation, but the videos I am looking for were curated for beginners, and we linked to them from our learning platform's Git and GitHub materials. Especially this playlist [Get Up and Running](https://www.youtube.com/watch?v=noZnOSpcjYY&list=PLg7s6cbtAD15G8lNyoaYDuKZSKyJrgwB-)
